WalletWasabi.Gui/Controls/WalletExplorer/SendTabView.xaml

```diff
@@ -41,7 +41,7 @@
           </controls:ExtendedTextBox>
 
           <controls:SuggestLabelView DataContext="{Binding LabelSuggestion}" />
-          
```

Comment:
Revert this formatting change

Reply:
Why should that be reverted?

Reply:
It's an unnecessary change and it interferes with git blame